src/source/feeds.rst

```diff
@@ -13,16 +13,63 @@ We treat the provided XML file as the desired list of ads the user wants to have
+:ref:`feed_vanityUrl`                  displayed URL                        max. 256 chars       no        
+:ref:`feed_priceType`                  sales model for product              enum                 yes       
+:ref:`feed_price`                      product price in cents if applicable positive integer     yes/no       
+:ref:`feed_originalPrice`              original price before discount       positive integer     no        
+:ref:`feed_media`                      product images                       :ref:`feed_media`    no
+:ref:`feed_attr`                       collection of ad attributes          :ref:`feed_attr`     no        
+:ref:`feed_budget`                     budget details                       :ref:`feed_budget`   no        
+:ref:`feed_ship`                       shipping options                     :ref:`feed_ship`     no
+:ref:`feed_phoneNumber`                phone number                         max. 32 chars        no        
+:ref:`feed_emailAdvertiser`            allow emails to the seller           true,false           no
+:ref:`feed_regionId`                   only applicable for Kijiji Canada    numeric              no        
+:ref:`feed_engines`                    targeted search engines for this ad  emunerated list      no
+:ref:`feed_microTip`                   tiny product hightlight              max. 18 chars        no
+:ref:`feed_mpn`                        Manufacturer Part Number (MPN)       len. 2-70 chars      no   
+:ref:`feed_googleProductCategory`      google category for your product     string               no
+:ref:`feed_productType`                customer product type                max. 750 chars       no    
+:ref:`feed_brand`                      product brand name                   max. 70 chars        no
+:ref:`feed_gtin`                       Global Trade Identification Number   max. 50 chars        no  
+:ref:`feed_itemGroupId`                groups product variants in your      len. 1-50 chars      no
+:ref:`feed_condition`                  condition of product                 enum                 no
+:ref:`feed_material`                   main product fabric or material      max. 200 chars       no
+:ref:`feed_energyEfficiencyClass`      energy efficiency class              enum                 no
+:ref:`feed_minEnergyEfficiencyClass`   minimal energy efficiency class      enum                 no
+:ref:`feed_maxEnergyEfficiencyClass`   maximal energy efficiency class      enum                 no
+:ref:`feed_color`                      product color                        len. 1-100 chars     no
+:ref:`feed_gender`                     gender product is designed for       enum                 no
+:ref:`feed_ageGroup`                   age group product is intended for    enum                 no
+:ref:`feed_size`                       size information                     enum                 no
```

Comment:
many of these fields are supported by the tenant's taxonomies; why do we add them here? 
The google feed is in CSV afaik, so why do we add these fields like this in the XML? (not flaming, just want to know)

Reply:
We are adding them as new, standalone fields for several reasons:
- We would like to move closer to industry standard in terms of how we capture product data from sellers, in order to to simplify the feed creation process and work towards a longer term goal of automating the category and attribute mapping process
- It also helps to slightly decouple ourselves and the sellers from the market-level taxonomy, which will likely be going through changes in the coming years due to Project Lighthouse 
- We have an assumption that these fields can generate significant value by being incorporated into search logic (e.g. keyword match in the "brand" field boosts ranking), which won't require any connection to market taxonomy, and a related assumption that market attributes don't provide much added value for either buyer or seller (for buyers the overall usage of attributes to refine search/browse results is very low across markets, for sellers the ROI of adding attributes is unclear and many aren't providing them)

And we are adding the fields to XML because we want to begin capturing these attributes from existing sellers who are providing their feeds via XML, as many of them already have much of these fields populated within feed management platforms they are using, we just need to provide the new fields to be mapped to.